library/core/src/ffi/c_str.rs

```diff
@@ -66,7 +66,7 @@ use crate::str;
 ///
 /// fn my_string_safe() -> String {
 ///     unsafe {
-///         CStr::from_ptr(my_string()).to_string_lossy().into_owned()
+///         String::from_utf8_lossy(CStr::from_ptr(my_string()))
```

Comment:
I think this won't typecheck, since `String::from_utf8_lossy` accepts a &[u8] and returns a `Cow`.

Reply:
Oh, good point!